packages/eas-cli/src/build/build.ts

```diff
@@ -385,6 +385,12 @@ async function handleSingleBuildProgressAsync(
   return { refetch: true };
 }
 
+const priorityToQueueDisplayName: Record<BuildPriority, string> = {
+  [BuildPriority.Normal]: 'Low Priority Queue',
+  [BuildPriority.NormalPlus]: 'High Priority Queue (free upgrade)',
+  [BuildPriority.High]: 'High Priority Queue',
```

Comment:
1. I'm not sure what looks better: uppercase or lowercase
2. I'm not sure if the wording for normal+ is correct. @brentvatne

Reply:
This is different wording than we use on the website ('Standard Queue', 'Priority Queue').

Reply:
Good point, sth like this?
```suggestion
  [BuildPriority.Normal]: 'Standard Queue',
  [BuildPriority.NormalPlus]: 'Priority Queue (free upgrade)',
  [BuildPriority.High]: 'Priority Queue',
```

Reply:
LGTM!

Reply:
```suggestion
  [BuildPriority.Normal]: 'queue',
  [BuildPriority.NormalPlus]: 'queue',
  [BuildPriority.High]: 'priority queue',
```

we don't want to surface the free upgrade bit at the moment, we just want those builds to be fast without that being advertised.

with the above suggestions, you'd see one of following two possible states:

> Waiting in queue...

> Waiting in priority queue...

(we don't use capitalized Priority on the pricing page)